### PR TITLE
fix(tab): group tab

### PR DESCRIPTION
**Pull request by hund030**
1. renaming the config file for consistency.
2. init teams sdk before configuration for ts template.

### CODE REVIEW

**Review by henzhang-ms — APPROVED** [review]

File: tab/ts/default/src/components/tabConfig.tsx [hunk 1]
```diff
@@ -14,6 +14,9 @@ import * as microsoftTeams from "@microsoft/teams-js";
 class TabConfig extends React.Component {
 
     render() {
```
Comment (henzhang-ms) [inline tab/ts/default/src/components/tabConfig.tsx:16]: Don't we need to update the file name?

Comment (henzhang-ms) [reply tab/ts/default/src/components/tabConfig.tsx:16]: ok, there are two tab config. One is ts and the other is tsx, is it fine?

Comment (hund030) [reply tab/ts/default/src/components/tabConfig.tsx:16]: Seems git ignore the renaming from tabConfig to TabConfig. How do I make it work?

Comment (hund030) [reply tab/ts/default/src/components/tabConfig.tsx:16]: Ok, I get it work with `git config core.ignorecase false`

**Pull request merged**